### DISCUSSION TITLE
LLDP doesn't have class 6 option to get 60watt power

### DISCUSSION THREAD

**Discussion by mmuazzam** [lldpd/lldpd#641] [Q&A]
Hi All,
I am trying to do lldp handshake with Juniper switch. but juniper switch is providing 0.0 watt. 
Previously we had configured "configure dot3 power pd type 1 class class-4 requested 51000 priority critical source pse supported enabled paircontrol powerpairs signal allocated 60000"

When we change command "configure dot3 power pd type 1 class class-4 requested **25000** priority critical source pse supported enabled paircontrol powerpairs signal allocated 60000"
Then switch gives power.

here we suspect class-4 has limitaion. Since class-6 supports power upto 90watt.

Do we have support to get 51watt from PSE device, do we have 3bt support.
If we have then what is the procedure to enable it.

**Comment by mmuazzam** [comment]
Hi @vincentbernat , can you please throw some light on this. We are unable to get power of 51watt from PSE device. Since class-4 is limited to 25watt only, but in lldpcli class-4 is the max and power type 2 is max.
Do we have some option so that we can use type 3 or 4 and class-6 ?

**Reply by vincentbernat** [reply]
802.3bt is half supported for now (no write support).

**Reply by mmuazzam** [reply]
@vincentbernat  so is there anyway we can configure or not at all.
what is "no write" means ?

**Reply by vincentbernat** [reply]
There is currently no way to generate LLDP packets with class 6. You can receive them (I think).

**Reply by mmuazzam** [reply]
Hi @vincentbernat 
Is there a way to use extended tlv kind of stuff to get 51watt. ?
